### ISSUE TITLE
Tip: alt text

### ISSUE THREAD

**Issue by slhenry** [w3c/wai-quick-start#151]
I'm not sure about the separation of writing and developing related to alt text. In some cases, is the writing task done by people who don't use authoring tools -- e.g., they provide the text in a word processing file, and then the developer puts it in the markup? If so, then alt text should be included in developing. I look forward to other EOWG perspectives on this. :)

There is a similar issue with page title -- crafting the wording is a writing task, but is including it in the markup a developing task?

**Comment by LydiaHarkey** [comment]
For the writing portion, I think it may help to have the writer determine if information is needed for the image and if so the writer will provide content. The Developer will need to be familiar with alt tag markup and collaborate with the writer on the content of the image.

**Comment by iadawn** [comment]
@LydiaHarkey Do you have a suggested edit to cover this?

Also, it is worth noting that this may stray too much into talking about how people perform the activities rather than the activities themselves.

**Comment by James-Green** [comment]
Thinking through this, I think we need alt text (and page title) tips in both places.

IMHO responsibility for alt text remains with whomever chose the image (maybe the writer, maybe a visual designer, maybe both together). This person(s) should be the best informed on what the page is trying to convey AND how users will interpret what they are shown.  For that same reason, the person(s) behind the inclusion of the page itself (maybe the content writer, maybe an information architect, maybe both together) should write the page titles and provide that to the developer.

**Comment by iadawn** [comment]
I have added a short sentence to highlight that alternative text usually comes from somewhere else. I hesitate to add role titles, such as content writers, simply because this is fraught with difficulties.

Closing issue. If there are still concerns, please do reopen.
